### PR TITLE
Fixed prop type for tabIndex

### PR DESCRIPTION
**Pull request by abramobagnara**

### CODE REVIEW

File: src/Checkbox.jsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ export default class Checkbox extends React.Component {
     onBlur: PropTypes.func,
     onChange: PropTypes.func,
     onClick: PropTypes.func,
-    tabIndex: PropTypes.string,
+    tabIndex: PropTypes.number,
```
Comment (yesmeck) [inline src/Checkbox.jsx]: For compatibility, it's better to keep `PropTypes.string`.

`PropTypes.oneOfType([PropTypes.string, PropTypes.number])`.

**Comment by abramobagnara** [comment]
Isn't better that legacy users get a proper warning?

**Comment by yesmeck** [comment]
`string` is also a valid type in React I think.

**Comment by abramobagnara** [comment]
I don't think this is true, please check yourself:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts

**Comment by abramobagnara** [comment]
Your proposal PropTypes.oneOfType([PropTypes.string, PropTypes.number]) would be fine for me, but this is not the path followed by rc-rate, rc-slider, rc-switch, etc.

I think it is better to decide what is better and be uniform.

**Comment by yesmeck** [comment]
Good point, we should uniform them to `PropTypes.oneOfType([PropTypes.string, PropTypes.number])`

**Pull request closed without merging**